src/Form/TimesheetEditForm.php

```diff
@@ -202,12 +203,12 @@ function (FormEvent $event) {
                 /** @var Timesheet $data */
                 $data = $event->getData();
                 $duration = $data->getDuration();
-                $end = null;
-                if (null !== $duration) {
+                if (!empty($duration)) {
```

Comment:
is this change necessary?

Reply:
Yep, although I'm naturally not sure if it's the right thing to do. Basically duration was somewhere reset to 0 if it was missing from the API which caused the data to be saved incorrectly; using empty() requires the value to be both not null and not zero.